_includes/scripts.html

```diff
@@ -58,54 +58,61 @@
   </menu>
   <div class="projects">
     <% let count=0 %>
-    <% _.each(projects, function(project) { %>
-    <% count= count+1%>
-    <div class="data-box" class="<% if ( project.stats ) { %>counted<% } %>">
-      <div>
-        <div colspan="2" class="title">
-           <span class="proj"><a href="<%-project.site %>" target="_blank"><%-project.name %></a></span>
-        </div>
-    </div>
-  <div>
-        <div class="details stats">
-        <p class="label">
-          <a href="<%- project.upforgrabs.link %>" title="View open issues for <%-project.name %>" target="_blank" tabindex="-1">
-            <%-project.upforgrabs.name %>
-            <% if (project.stats && project.stats['last-updated']) {
-                const now = new Date();
-                let lastUpdated = new Date(project.stats['last-updated']);
-                let since = relativeTime(now, lastUpdated);
-              %>
-              <span class="count" title="Last updated <%- since %>"><%-project.stats['issue-count']%></span>
-              <% } else if (project.stats) { %>
-              <span class="count" title="No recent activity"><%-project.stats['issue-count']%></span>
-            <% } %>
-          </a>
-        </p>
-        <% if (project.stats) { %>
-            <div class="fork-count">
-              <svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true" class="fork-count-svg">
-                <path d="M5 5.372v.878c0 .414.336.75.75.75h4.5a.75.75 0 0 0 .75-.75v-.878a2.25 2.25 0 1 1 1.5 0v.878a2.25 2.25 0 0 1-2.25 2.25h-1.5v2.128a2.251 2.251 0 1 1-1.5 0V8.5h-1.5A2.25 2.25 0 0 1 3.5 6.25v-.878a2.25 2.25 0 1 1 1.5 0ZM5 3.25a.75.75 0 1 0-1.5 0 .75.75 0 0 0 1.5 0Zm6.75.75a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5Zm-3 8.75a.75.75 0 1 0-1.5 0 .75.75 0 0 0 1.5 0Z"></path>
-              </svg>
-              <%-project.stats['fork-count']%>
+      <% _.each(projects, function(project) { %>
+        <% count=count+1%>
+          <div class="data-box" class="<% if ( project.stats ) { %>counted<% } %>">
+            <div>
+              <div>
+                <div colspan="2" class="title">
+                  <span class="proj"><a href="<%-project.site %>" target="_blank"><%-project.name %></a></span>
+                </div>
+              </div>
+              <div>
+                <div class="details stats">
+                  <p class="label">
+                    <a href="<%- project.upforgrabs.link %>" title="View open issues for <%-project.name %>"
+                      target="_blank" tabindex="-1">
+                      <%-project.upforgrabs.name %>
+                        <% if (project.stats && project.stats['last-updated']) { const now=new Date(); let lastUpdated=new
+                          Date(project.stats['last-updated']); let since=relativeTime(now, lastUpdated); %>
```

Comment:
This compression of multiple lines also feels weird - can we bring back the previous formatting and make this code easier to follow?

Reply:
Is it good now ?

Reply:
No - see how the previous style broke the code over multiple lines?

```js
 <% if (project.stats && project.stats['last-updated']) {
                const now = new Date();
                let lastUpdated = new Date(project.stats['last-updated']);
                let since = relativeTime(now, lastUpdated);
              %>
```

That context is lost here

Reply:
Sorry for the fault , but I have fixed this issue now.